xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
-   <calloutlist>
-    <callout arearefs="co-ldap-server-dsrc-remote-localhost">
-     <para>
-      Needs to point to the LDAP server instance. If not specified otherwise,
-      the default instance name is <literal>localhost</literal>.
-     </para>
-    </callout>
-    <callout arearefs="co-ldap-server-dsrc-remote-cert">
-     <para>
-      Path to the certificate, at a readable location or on the client machine, from
-      which you use the <command>ds*</command> commands.
-     </para>
-    </callout>
-   </calloutlist>
-  </example>
-  <para>
-   If you want to administer the instance on the same host where the &ds389; runs,
-   use the configuration file in <xref linkend="ex-security-ldap-server-credentials-local"
-   xrefstyle="select:label"/>.
-  </para>
-  <example xml:id="ex-security-ldap-server-credentials-local">
-   <title>A <filename>.dsrc</filename> File for Local Administration</title>
-   <screen># cat ~/.dsrc
-[localhost]
-# Note that '/' is replaced with '%%2f'.
-uri = ldapi://%%2fvar%%2frun%%2fslapd-localhost.socket <co xml:id="co-ldap-server-dsrc-remote-ldapi"/>
-basedn = dc=example,dc=com
+uri = ldapi://<replaceable>%%2fvar%%2frun%%2fslapd-389ds-test.socket</replaceable> <co xml:id="co-ldap-server-dsrc-remote-ldapi"/>
+basedn = <replaceable>dc=test1,dc=com</replaceable>
 binddn = cn=Directory Manager
 </screen>
    <calloutlist>
+    <callout arearefs="co-ldap-server-dsrc-instance-name">
+      <para>
+        This must specify the instance name that you created in your
+        <filename>.inf</filename> file.
+      </para>
+    </callout>       
     <callout arearefs="co-ldap-server-dsrc-remote-ldapi">
       <para>
-       <!--Question: “When I use ldapi on the server that has the DS instance,
-       why don’t I need to provide my password?”--> When using <literal>ldapi</literal>
-       on the server where the &ds389; instance is running, your UID/GID will be
-       detected. If it is <literal>0/0</literal> (which means you are logged
-       in as &rootuser; user), the <literal>ldapi</literal> binds the local
-       &rootuser; as the directory server root dn (<literal>cn=Directory Manager</literal>)
-       of the instance. This allows local administration of the server, but also
-       allows you to set a machine-generated password for <literal>cn=Directory
-       Manager</literal> that no human knows. Whoever has administrator rights
-       on the server hosting the &ds389; instance can access the instance as
-       <literal>cn=Directory Manager</literal>.
+        <literal>ldapi</literal> detects the UID and GID of the user attempting 
+        to log in to the server. If the UID/GID are <literal>0/0</literal>,
+        <literal>ldapi</literal> binds the system root user (or sudo user) as 
+        the directory server root dn, which is 
+        <literal>cn=Directory Manager</literal>. If the UID/GID are not 
+        <literal>0/0</literal> then the user is not allowed to log into the server.
```

Comment:
If the uid/gid aren't 0/0 then we trigger the ldapi uid/map and it looks up your uidNumber in the directory AND autobinds you to that. So this is not necessarily correct .... It may just be that your "uid/gid" don't have a corresponding entry sou that's why the bind fails.

Reply:
aha. I was thinking there should be a way for other users to log in. I will come back to this.

Reply:
Other admin users, that is.